webapp/src/contexts/ConferenceContext/ConferenceContext.tsx

```diff
@@ -81,6 +82,18 @@ const ConferenceContextProvider = (props: any): JSX.Element => {
       .catch(console.error)
   }
 
+  useEffect(() => {
+    ;(window as any).desktopAPI
```

Comment:
you have unnecessary `;` in there

Reply:
It's added automatically by the linting and applied when saved. I assume there is a rule for that we could disable.